src/components/panel/panel.js

```diff
@@ -806,6 +860,62 @@ MdPanelService.prototype.newPanelAnimation = function() {
 
 
 /**
+ * Creates a panel group and adds it to a tracked list of panel groups.
+ * @param groupName {string} Name of the group to create.
+ * @param config {!Object=} Specific configuration object that may contain the
+ *     following properties:
+ *
+ *   - `maxOpen` - `{number=}`: The maximum number of panels that are allowed
+ *     open within a defined panel group.
+ */
+MdPanelService.prototype.newPanelGroup = function(groupName, config) {
+  if (!this._groups.hasOwnProperty(groupName)) {
+    config = config || {};
+    var group = {
+      panels: [],
+      openPanels: [],
+      maxOpen: config.maxOpen > 0 ? config.maxOpen : false
```

Comment:
Another question: What do you think about having the default value for `maxOpen` be `Infinity`. This could spare us some issues when comparing numbers to `false`.